mysql-test/t/plugin-load-add-with-path.test

```diff
@@ -0,0 +1,107 @@
+# Test for ability to use path in --plugin-load and --plugin-load-add options
+
+#
+# Check if server has support for loading plugins
+#
+if (`SELECT @@have_dynamic_loading != 'YES'`) {
+  --skip This test requires dynamic loading
+}
+
+#
+# Check if the variable EXAMPLE_PLUGIN is set
```

Comment:
It does not suite because have_example_plugin.inc checks if plugin_var variable contains the plugin dir, what is wrong for our case. The correspondent comment is added in the *.test file.